### PR TITLE
[s] COVER ME, I'M MAKING AN APRIL FOOLS PR!!

### PR DESCRIPTION
**Pull request by PKPenguin321**
# REEEEELOADING!!

### CODE REVIEW

**Comment by duncathan** [comment]
YES

**Comment by Bawhoppen** [comment]
perfect thing to do

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 1]
```diff
@@ -36,6 +46,8 @@
 		var/obj/item/ammo_box/magazine/AM = A
 		if(istype(AM, mag_type))
 			if(magazine)
+				if(tactical)
+				user.say(pick("I'M RELOADING!!!", "NEW MAG IN!!!", "CHANGING MAG!!!", "COVER ME! I'M RELOADING!!!", "RELOADING!!!"))
```
Comment (Iamgoofball) [inline code/modules/projectiles/guns/projectile/automatic.dm]: forgot a tab here

**Comment by Xhuis** [comment]
nice commits

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/modules/projectiles/guns/projectile/automatic.dm]: fugg

**Comment by PKPenguin321** [comment]
>nice commits

It's supposed to do that.

**Comment by Supermichael777** [comment]
but im already puting something on altclick. this will make it so you only say this with the safety on. also you should target the april fools branch in your fork and then do your change slecting to make a new branch.

**Comment by Bawhoppen** [comment]
no the commits keep the branch up to date

**Comment by PKPenguin321** [comment]
>but im already puting something on altclick. this will make it so you only say this with the safety on. also you should target the april fools branch in your fork and then do your change slecting to make a new branch.

uuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuugh
ffiiiine ill redo this later

**Comment by duncathan** [comment]
this PR is way better than dumb safeties

**Pull request closed without merging**